### PR TITLE
grafana/toolkit: Update FAQ

### PR DESCRIPTION
**Pull request by dprokop**
Adds information about dealing with `Error: Unable to dynamically transpile ES module A loader plugin needs to be configured` SystemJS error

### CODE REVIEW

File: packages/grafana-toolkit/README.md [hunk 1]
```diff
@@ -216,6 +216,9 @@ module.exports = {
 ### How do I add 3rd party dependencies that are not npm packages?
```
Comment (oddlittlebird) [inline packages/grafana-toolkit/README.md]:
```suggestion
### How do I add third-party dependencies that are not npm packages?
```

File: packages/grafana-toolkit/README.md [hunk 2]
```diff
@@ -216,6 +216,9 @@ module.exports = {
...
 You can add such dependencies by putting them in `static` directory in the root of your project. The `static` directory will be copied when building the plugin.
```
Comment (oddlittlebird) [inline packages/grafana-toolkit/README.md]:
```suggestion
Put them in the `static` directory in the root of your project. The `static` directory is copied when the plugin is built.
```

File: packages/grafana-toolkit/README.md [hunk 3]
```diff
@@ -216,6 +216,9 @@ module.exports = {
...
 
+### I have encountered `Unable to dynamically transpile ES module A loader plugin needs to be configured via SystemJS.config({ transpiler: 'transpiler-module' }).`. What should I do?
```
Comment (oddlittlebird) [inline packages/grafana-toolkit/README.md]:
```suggestion
### I am getting this message: `Unable to dynamically transpile ES module A loader plugin needs to be configured via SystemJS.config({ transpiler: 'transpiler-module' }).`
```

Comment (oddlittlebird) [reply packages/grafana-toolkit/README.md]: When would they be getting the message? During build? Another time?

File: packages/grafana-toolkit/README.md [hunk 4]
```diff
@@ -216,6 +216,9 @@ module.exports = {
...
+This error can occur when you bundle your plugin using `grafana-toolkit plugin:dev` task. This task does not remove comments from the bundled package, so **make sure your comments does not include ES2016 code**. If they do, SystemJS v0.20.19 which Grafana uses internally to load plugins, will interprete your code as an [ES module](https://hacks.mozilla.org/2018/03/es-modules-a-cartoon-deep-dive/). Grafana does not support ESM.
```
Comment (oddlittlebird) [inline packages/grafana-toolkit/README.md]:
```suggestion
This error occurs when you bundle your plugin using the `grafana-toolkit plugin:dev` task and your code comments include ES2016 code.

There are two issues at play:
* The `grafana-toolkit plugin:dev` task does not remove comments from your bundled package.
* Grafana does not support ESM.

If your comments include ES2016 code, then SystemJS v0.20.19, which Grafana uses internally to load plugins, interprets your code as an [ES module](https://hacks.mozilla.org/2018/03/es-modules-a-cartoon-deep-dive/) and fails.

To fix this error, remove the ES2016 code from your comments.
```

Comment (oddlittlebird) [reply packages/grafana-toolkit/README.md]: Please spell out what ES stands foron the first usage.

File: packages/grafana-toolkit/README.md [hunk 5]
```diff
@@ -216,6 +216,9 @@ module.exports = {
...
+
 ## Contribute to grafana-toolkit
 You can contribute to grafana-toolkit in the by helping develop it or by debugging it.
```
Comment (oddlittlebird) [inline packages/grafana-toolkit/README.md]:
```suggestion
You can contribute to grafana-toolkit by helping develop it or by debugging it.
```

Comment (oddlittlebird) [reply packages/grafana-toolkit/README.md]: Please add a link to the appropriate document or documents in the `contribute` folder.

**Review by hugohaggmark — APPROVED** [review]
LGTM!

**Comment by smarkm** [comment]
I got this tip from https://fossies.org/linux/grafana/packages/grafana-toolkit/README.md suggest to " remove the ES2016 code from your comments", is there another way to allow load ES module? the `plugin:build`  can work well, so should be have another way to allow it?

**Comment by oddlittlebird** [comment]
Please post a question on community.grafana.com or open an issue. Closed PRs are not the appropriate place to ask questions.

**Pull request merged**